test/e2e_node/image_list.go

```diff
@@ -43,9 +43,9 @@ const (
 	imagePullRetryDelay = time.Second
 )
 
-// NodeImageWhiteList is a list of images used in node e2e test. These images will be prepulled
+// NodeImageAllowList is a list of images used in node e2e test. These images will be prepulled
```

Comment:
mm I think this may be similar to https://github.com/kubernetes/kubernetes/pull/92318.
Seems to be that "white" is not used as a filter but to convey that the set of images is going to be pre-pulled.
So may be rename it to `NodePrePullImageList` or something like that?